payjoin/src/send/mod.rs

```diff
@@ -11,129 +14,6 @@
 //! 6. Process the response with [`Context::process_response()`](crate::send::Context::process_response())
 //! 7. Sign and finalize the Payjoin Proposal PSBT
 //! 8. Broadcast the Payjoin Transaction (and cancel the optional fallback broadcast)
-//!
-//! This crate is runtime-agnostic. Data persistence, chain interactions, and networking may be provided by custom implementations or copy the reference [`payjoin-cli`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-cli) for bitcoind, [`nolooking`](https://github.com/chaincase-app/nolooking) for LND, or [`bitmask-core`](https://github.com/diba-io/bitmask-core) BDK integration. Bring your own wallet and http client.
```

Comment:
removed this paragraph as in the `lib.rs` its mentioned that its io agnostic and so on. but the examples are not mentioned. I think though that those examples should be in the github readme rather than in the documentation

Reply:
If you think they should be in the readme instead, propose a single commit/pr that moves them from there to here. Examples are in docs because they were specifically asked for from downstream users (bullbitcoin) Removing them without replacing elsewhere is lost documentation. I appreciate the proactivity but jumbling together change proposals makes them much more difficult to review and blocks progress merging the uncontroversial, already discussed changes